Cargo.toml

```diff
@@ -1,18 +1,19 @@
 [package]
```

Comment:
I think we should have a 2 -level cargo.toml with workspace dependencies

Reply:
This repo is currently a single package